src/ublue_update/cli.py

```diff
@@ -6,6 +6,31 @@
 import tomllib
 import argparse
 
+from update_checks.system import system_update_check
+
+
+def check_for_updates(checks_failed: bool):
+    """Tracks whether any updates are available"""
+    update_available = False
+    system_update_available = system_update_check()
+    if system_update_available:
+       update_available = True
+    if update_available:
+        """Only display override notification if checks failed"""
+        if dbus_notify and checks_failed:
+            update_notif = notify2.Notification(
+                "System Updater",
+                f"Update available, but system checks failed. Update now?",
+                "notification-message-im")
+            update_notif.add_action(
+                "update",
+                "Update",
+                run_updates()
+            )
+            update_notif.show()
+        return 0
+    raise Exception(f"No updates available.")
```

Comment:
I think it would probably be better to use exit() with an exit code rather than raising an exception, and before the exit logging the error